airflow/providers/cncf/kubernetes/utils/pod_launcher.py

```diff
@@ -320,20 +332,3 @@ def _exec_pod_command(self, resp, command: str) -> None:
                     self.log.info(resp.read_stderr())
```

Comment:
Should we say what this is?

Reply:
say what _what_ is?

Reply:
Something like:

```suggestion
                    self.log.info("stderr from command: %s", resp.read_stderr())
```

In fact, should we log stderr before we return stdout? We could have both...

Reply:
ah yes, thanks

yeah it's too bad they are not merged... isn't it possible that they could have interleaved timing, if that makes sense?  can  take a look at that

Reply:
looked a little more closely and understand a little better what this is doing

this is actually a bit weird.  it looks very similar to this example here: https://github.com/kubernetes-client/python/blob/master/examples/pod_exec.py#L96-L107

but what that example is doing is looping through the log stream and emitting stdout / stderr from each chunk as it goes.

what we are doing here _looks_ very similar.  but we seem to return the first time _anything_ goes to the logs.

Reply:
update -- this is only used for xcom retrieval and killing of xcom container